src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -483,6 +598,19 @@ class ExplorerButtonGroup extends React.Component {
       // disable the pfb export button if any other pfb export jobs are running
       return !(this.state.exportingToTerra || this.state.exportingToSevenBridges);
     }
+    if (buttonConfig.type === 'export-files-to-pfb') {
+      // disable the pfb export button if any other pfb export jobs are running
+      const otherPFBJobsRunning = this.state.exportingToTerra
+        || this.state.exportingToSevenBridges;
```

Comment:
why we don't check for `isPFBRunning()` in here (also in `export-to-pfb`)?

and there are so many trios of `this.state.exportingToTerra || this.state.exportingToSevenBridges || this.isPFBRunning()`, maybe we should consolidate them 🤔

Reply:
> why we don't check for isPFBRunning() in here (also in export-to-pfb)?
I wasn't sure why and I was scared to change it 😆. But I can't imagine how adding that extra check for `isPFBRunning()` would break anything, so I'll consolidate the check for `this.state.exportingToTerra || this.state.exportingToSevenBridges || this.isPFBRunning()` into a single function.